exercises/supervised-classification/ex_rnw/ex_mlr_dec_boundaries.Rnw

```diff
@@ -1,3 +1,19 @@
-Choose some of the classifiers already introduced in the lecture and visualize their decision boundaries for relevant hyperparameters.
-Use \texttt{mlbench::mlbench.spirals} to generate data and use \texttt{plot\_learner\_prediction} for visualization. To refresh your knowledge about \texttt{mlr3} you can take a look at \url{https://mlr3book.mlr-org.com/basics.html}.
+We will now visualize how well different learners classify the notoriously 
```

Comment:
Zu dem Zeitpunkt haben sie LDA, QDA und NB noch nicht gehört - schieben wir das in Blatt 2 der Klassifikation? Also ich würde sagen ja - außer es tut dort sehr weh?